packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -217,45 +173,30 @@ export async function activate(context: vscode.ExtensionContext) {
   telemetryService.sendExtensionActivationEvent();
 
   extContext = context;
-
-  // registerCommands needs the checkpoint configuration
-  const config = vscode.workspace.getConfiguration();
-  const checkpointsEnabled = config.get(
-    'salesforcedx-vscode-apex-replay-debugger-checkpoints.enabled',
-    false
-  );
-
-  vscode.commands.executeCommand(
-    'setContext',
-    'sfdx:replay_debugger_checkpoints_enabled',
-    checkpointsEnabled
-  );
-
-  const commands = registerCommands(checkpointsEnabled);
+  const commands = registerCommands();
   const debugHandlers = registerDebugHandlers();
   const debugConfigProvider = vscode.debug.registerDebugConfigurationProvider(
     'apex-replay',
     new DebugConfigurationProvider()
   );
-  context.subscriptions.push(commands, debugHandlers, debugConfigProvider);
-
-  // Don't create the checkpoint service or register the breakpoints event
-  // if checkpoints aren't enabled
-  if (checkpointsEnabled) {
-    const checkpointsView = vscode.window.registerTreeDataProvider(
-      'sfdx.force.view.checkpoint',
-      checkpointService
-    );
-    context.subscriptions.push(checkpointsView);
+  const checkpointsView = vscode.window.registerTreeDataProvider(
+    'sfdx.force.view.checkpoint',
+    checkpointService
+  );
+  const breakpointsSub = vscode.debug.onDidChangeBreakpoints(
+    processBreakpointChangedForCheckpoints
+  );
 
-    const breakpointsSub = vscode.debug.onDidChangeBreakpoints(
-      processBreakpointChangedForCheckpoints
-    );
-    context.subscriptions.push(breakpointsSub);
-    console.log(
-      'in activate, added breakpointsSub to subscriptions, activation complete'
-    );
-  }
+  context.subscriptions.push(
+    commands,
+    debugHandlers,
+    debugConfigProvider,
+    checkpointsView,
+    breakpointsSub
+  );
+  console.log(
+    'in activate, added breakpointsSub to subscriptions, activation complete'
```

Comment:
@lcampos - Do we still need this logging here? It's a bit noisy and I would like to remove it if possible.

Reply:
@lcampos and @vazexqi that log line can definitely be removed.